src/test/java/com/atlassian/bitbucket/jenkins/internal/util/TestUtils.java

```diff
@@ -45,7 +45,8 @@ public static String encode(String urlSnippet) {
     public static String readFileToString(String relativeFilename) {
         try {
             return new String(
-                    readAllBytes(Paths.get(TestUtils.class.getResource(relativeFilename).toURI())));
```

Comment:
I think we should now add a test for this, it is very small and easy but regular expressions are very error prone.